### PR TITLE
Hacktoberfest

### PR DESCRIPTION
**Pull request by ruskibenya**
## Description

Updates to the /devtv and /hacktoberfest landing pages. Some image/description updates to devtv and content updates to Hacktoberfest

### CODE REVIEW

**Comment by ruskibenya** [comment]
Testing for avital

**Review by avital-vonage — CHANGES_REQUESTED** [review]

File: custom/landing_pages/devtv.yml [hunk 1]
```diff
@@ -732,6 +732,9 @@ page:
                         <h5>
                           Java Developer Advocate
                         </h5>
+                        <p class="text-justify">
+                          Yvonne is an Angular fullstack developer , Java developer, and speaker. Yvonne is currently the Java Developer Advocate at Vonage, previously known as Nexmo. She enjoys working with the tech community and helping mentor others who aim to advance in their tech career.
```
Comment (avital-vonage) [inline custom/landing_pages/devtv.yml]:
```suggestion
                          Yvonne is an Angular fullstack developer, Java developer, and speaker. Yvonne is currently the Java Developer Advocate at Vonage, previously known as Nexmo. She enjoys working with the tech community and helping mentor others who aim to advance in their tech career.
```

**Review by avital-vonage — COMMENTED** [review]
Yvonne's bio

**Review by avital-vonage — APPROVED** [review]
Looks great!

**Review by avital-vonage — APPROVED** [review]

**Pull request merged**